cmd/usage.js

```diff
@@ -94,29 +94,30 @@ module.exports = ({ fork, length, key }) => {
   `
 
   const run = ansi.bold(cmd + ' run')
-  const runArgs = ansi.bold('<key|dir|alias> [...args]')
-  const runBrief = 'Run an application from a key or dir.'
+  const runArgs = ansi.bold('[flags] <key|dir|alias> [...app-args]')
+  const runBrief = 'Run an application from a key or dir. app-args are passed directly to the app.'
```

Comment:
this is displayed in main help so it needs to be brief (as per the name) 

You can put this explanation in runExplain though